state/src/interface/wallet_index.rs

```diff
@@ -126,7 +126,7 @@ mod test {
             .map(|_| OrderMetadata {
                 id: OrderIdentifier::new_v4(),
                 state: OrderState::Filled,
-                filled: 1,
+                fills: vec![],
```

Comment:
Should this have a partial fill w amount 1? If tests pass tho ofc doesn't matter